public/js/track.js

```diff
@@ -309,7 +340,7 @@ function sendPOSTRequestToCreatePlaylist(data) {
             if (response && response.data && response.data.link && typeof response.data.link === 'string') {
                 return response.data.link;
             } else {
-                throw new Error("response is null or response.link is null or not a string");
+                throw new Error("Response is null or response.link is null or not a string");
```

Comment:
Will this be an alert?

Reply:
Yup, all error messages are passed into the alert

Reply:
Ah okie sorry i misread, thanks!